### DISCUSSION TITLE
蓝奏云增强展示

### DISCUSSION THREAD

**Discussion by ghost** [AlistGo/alist#192] [Ideas]
蓝奏云还是很好用的，只是只能分享一个目录。
能不能添加一个约定标记，譬如book-123.doc就生成目录列为book/123.doc。这钟增强是可选的，可以让使用者自行选择是否增强。

**Comment by xhofe** [comment]
需要多文件夹可以使用cookie挂载，只是需要定期更新。

**Comment by ghost** [comment]
那考不考虑一个设计一个定期自动登录获取cookie的脚本啊

**Reply by xhofe** [reply]
蓝奏云有阿里的滑动验证 没法破解只能通过selenium模拟登录来获取cookie 
暂时不想写 

可以尝试自己写一下

**Reply by ghost** [reply]
奥奥，明白了。谢谢
